extensions/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN --mount=type=secret,id=yumrepos,target=/os/secret.repo extensions/build.sh
 ## Creates the repo metadata for the extensions.
 ## This uses Fedora as a lowest-common-denominator because it will work on
 ## current p8/s390x. See https://github.com/openshift/os/issues/1000
-FROM quay.io/fedora/fedora:41 as builder
+FROM quay.io/fedora/fedora:42 as builder
 COPY --from=os /usr/share/rpm-ostree/extensions/ /usr/share/rpm-ostree/extensions/
 RUN rm -f /etc/yum.repos.d/*.repo \
 && curl -L https://raw.githubusercontent.com/coreos/fedora-coreos-config/testing-devel/fedora-archive.repo -o /etc/yum.repos.d/fedora-archive.repo
```

Comment:
I don't understand why we are pulling in the archive repo here. This should probably be dropped.

Reply:
We'll still need to keep this while we are building on the ITUP cluster: https://github.com/openshift/os/pull/1571/commits/8301c67176e71ff2c6c5236109c89a149422a592

Reply:
https://github.com/openshift/os/pull/1603#issuecomment-2840071802